docs/source/conf.py

```diff
@@ -573,9 +572,7 @@ def adds(pth):
 """
 
 # add link to original notebook at the bottom
-notebook_url = (
-    f"https://github.com/sktime/sktime/tree/{CURRENT_VERSION}/{current_file}"  # noqa
-)
+notebook_url = f"https://github.com/sktime/sktime/tree/{CURRENT_VERSION}/{current_file}"  # noqa
```

Comment:
I think we should not use a broad `noqa` unless it's totally necessary.

What is the issue here, line length?

Reply:
these were mostly introduced before these changes but I am going to go through the codebase and resolve that